src/tribol/mesh/CouplingScheme.hpp

```diff
@@ -832,6 +839,8 @@ class CouplingScheme {
 
   PairReportingData m_pairReportingData;  ///< struct handling on-rank pair reporting data from computational geometry
 
+  RealT m_effective_binning_proximity_scale;  ///< Binning proximity scaled by the LOR factor
```

Comment:
Minor: consider adding note as to why we do this. Scaling by the LOR factor makes the proximity detection independent of the LO refinement so you get consistent RCB, binning, geom filter for a fixed HO mesh regardless of LOR factor. I can envision forgetting this unless it is written down

Reply:
Added